src/cli/project/export/pypi_requirements.rs

```diff
@@ -0,0 +1,256 @@
+use std::collections::HashSet;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+use clap::Parser;
+use miette::{Context, IntoDiagnostic};
+
+use crate::cli::cli_config::PrefixUpdateConfig;
+use crate::cli::LockFileUsageArgs;
+use crate::lock_file::UpdateLockFileOptions;
+use crate::Project;
+use rattler_conda_types::Platform;
+use rattler_lock::{Environment, Package, PackageHashes, PypiPackage, PypiPackageData, UrlOrPath};
+
+#[derive(Debug, Parser)]
+#[clap(arg_required_else_help = false)]
+pub struct Args {
+    /// Output directory for rendered requirements files
+    pub output_dir: PathBuf,
+
+    /// Environment to render. Can be repeated for multiple envs. Defaults to all environments
+    #[arg(short, long)]
+    pub environment: Option<Vec<String>>,
+
+    /// The platform to render. Can be repeated for multiple platforms.
+    /// Defaults to all platforms available for selected environments.
+    #[arg(short, long)]
+    pub platform: Option<Vec<Platform>>,
+
+    /// Conda dependencies are not supported in pip requirements files.
+    /// This flag allows creating the requirements file even if Conda dependencies are present.
+    #[arg(long, default_value = "false")]
+    pub ignore_conda_errors: bool,
```

Comment:
You cannot have a pixi project without conda-dependencies if their are pypi dependencies. So this is a mandatory flag. This logic should be removed.

Reply:
Sounds good. I'll try to make the changes you requested shortly and will hopefully have everything ready today